providers/amazon/src/airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -145,7 +145,7 @@ def create_glue_job_config(self) -> dict:
 
         return config
 
-    def list_jobs(self) -> list:
+    def describe_jobs(self) -> list:
```

Comment:
That's a breaking change. We should deprecate `list_jobs`, create a copy of it and name it `describe_jobs` and call the latter in `list_jobs`